### PR TITLE
Migrate nomad backend plugin to new backend system

### PR DESCRIPTION
**Pull request by lavanya-sainik-ericsson**
#### :heavy_check_mark: Checklist

- [x] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [x] Added or updated documentation
- [x] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)
- [x] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

**Comment by benjdlambert** [comment]
hey @lavanya-sainik-ericsson :wave:

Looks like you'll need to update the api-reports here so that we can see what's happened! 

`yarn buiild:api-reports` should generate you some new `markdown` files that will need pushing to this branch in order to make the build green :tada:

**Comment by lavanya-sainik-ericsson** [comment]
Hi [benjdlambert](https://github.com/benjdlambert),
Thanks for your comment. I have generated the api-report. Please have a look and let me know if any more comments / corrections needed.

File: plugins/nomad-backend/README.md [hunk 1]
```diff
@@ -1,6 +1,21 @@
...
+The Nomad backend plugin has support for the [new backend system](https://backstage.io/docs/backend-system/), here's how you can set that up:
+
+In your `packages/backend/src/index.ts` make the following changes:
+
+```diff
+  import { createBackend } from '@backstage/backend-defaults';
+  import { nomadPlugin } from '@backstage/plugin-nomad-backend';
+  const backend = createBackend();
+  // ... other feature additions
++ backend.add(nomadPlugin());
```
Comment (benjdlambert) [inline plugins/nomad-backend/README.md:16]: I think that these docs can be replaced with the example above:

```
backend.add(import('...'))
```

instead :)

**Review by benjdlambert — CHANGES_REQUESTED** [review]
Just a quick comment on the docs :)

**Comment by lavanya-sainik-ericsson** [comment]
Hey [benjdlambert](https://github.com/benjdlambert), 

Thanks for your comment. Done that change :) Please have a look and let me know if any more comments / corrections needed.

**Comment by lavanya-sainik-ericsson** [comment]
Hey [benjdlambert](https://github.com/benjdlambert),

Thanks  :) Please have a look again and let me know if any more comments / corrections needed.

**Review by benjdlambert — APPROVED** [review]
Nice thanks :tada:

**Pull request merged**